code/modules/gear_presets/uscm_ship.dm

```diff
@@ -534,7 +535,7 @@
 	name = "USCM Commanding Officer (CO++)"
 	idtype = /obj/item/card/id/general
 	paygrades = list(PAY_SHORT_MO6 = JOB_PLAYTIME_TIER_0)
-	dress_under = list(/obj/item/clothing/under/marine/dress/blues/senior, /obj/item/clothing/under/marine/dress/blues/general)
+	dress_under = list(/obj/item/clothing/under/marine/dress/blues/senior, /obj/item/clothing/under/marine/dress/blues/general, /obj/item/clothing/suit/storage/jacket/marine/dress)
```

Comment:
jacket should be in `dress_over` instead of `dress_under` here

Reply:
oh, my bad. I will fix it

Reply:
```suggestion
	dress_under = list(/obj/item/clothing/under/marine/dress/blues/senior, /obj/item/clothing/under/marine/dress/blues/general)
```